crates/net/rpc-api/src/admin.rs

```diff
@@ -14,6 +14,17 @@ pub trait AdminApi {
     #[method(name = "admin_removePeer")]
     async fn remove_peer(&self, record: String) -> Result<bool>;
 
+    /// Adds the given node record to the trusted peerset.
+    #[method(name = "admin_addTrustedPeer")]
+    async fn add_trusted_peer(&self, record: String) -> Result<bool>;
+
+    /// Removes a remote node from the trusted peer set, but it does not disconnect it
+    /// automatically.
+    ///
+    /// Returns true if the peer was successfully removed.
+    #[method(name = "admin_removeTrustedPeer")]
+    async fn remove_trusted_peer(&self, record: String) -> Result<bool>;
```

Comment:
@mattsse `record` will be parsed into a `NodeRecord` type in the impl, right? looked thru `jsonrpsee` crate, but couldn't find anything that would allow for the custom deser, that's why it's a `String`?

Reply:
geth also accepts strings here, the deserde of NodeRecord should be fromstr now, but we can migrate this eventually for all calls